plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/CacheMediatorDeserializer.java

```diff
@@ -59,26 +60,27 @@ public CacheMediator createNode(IGraphicalEditPart part, AbstractMediator object
 			if (mediator.isCollector()) {
 				executeSetValueCommand(CACHE_MEDIATOR__CACHE_ACTION, CacheAction.COLLECTOR);
```

Comment:
Formatting issues are fixed in : https://github.com/wso2/devstudio-tooling-esb/pull/272